package.json

```
@@ -3,11 +3,14 @@
     "version": "1.0.4",
     "private": true,
     "scripts": {
-        "prestart": "yarn run buildplugin && yarn link docusaurus-plugin-navdata",
-        "buildplugin": "cd ./src/plugins/docusaurus-plugin-navdata && yarn install && yarn link",
+        "prestart": "yarn run buildnavplugin && yarn link docusaurus-plugin-navdata && yarn run buildcookiesplugin && yarn link docusaurus-plugin-cookiesbanner",
+        "unlink-nav": "cd ./src/plugins/docusaurus-plugin-navdata && yarn unlink && cd ./src/plugins/docusaurus-plugin-cookiesbanner && yarn unlink",
+        "unlink-banner": "cd ./src/plugins/docusaurus-plugin-cookiesbanner && yarn unlink",
+        "buildnavplugin": "cd ./src/plugins/docusaurus-plugin-navdata && yarn install && yarn link",
+        "buildcookiesplugin": "cd ./src/plugins/docusaurus-plugin-cookiesbanner && yarn install && yarn link",
```

@msalmeron What about silent package `unlink` attempt, just before link?

```suggestion
        "buildnavplugin": "cd ./src/plugins/docusaurus-plugin-navdata && yarn install && (yarn unlink > /dev/null || true) && yarn link",
        "buildcookiesplugin": "cd ./src/plugins/docusaurus-plugin-cookiesbanner && yarn install && (yarn unlink > /dev/null || true) && yarn link",
```

This would prevent those `link` warnings.

---

I like it, this should work. Will run some tests to double check that everything works as expected